minikube default installation using Docker can be used for TiDB (#1544)
pingcap/docs-tidb-operator: en/get-started.md

You can create a Kubernetes cluster in a VM using [minikube](https://minikube.sigs.k8s.io/docs/start/), which supports macOS, Linux, and Windows.

Before deployment, make sure the following requirements are satisfied:

> - [minikube](https://minikube.sigs.k8s.io/docs/start/): version 1.0.0 or later versions. We recommend upgrading to the newer versions like v1.24. minikube requires a compatible hypervisor. For details, refer to minikube installation instructions.

Reviewer: ```suggestion
- [minikube](https://minikube.sigs.k8s.io/docs/start/): version 1.0.0 or later versions. Newer versions like v1.24 is recommended. minikube requires a compatible hypervisor. For details, refer to minikube installation instructions.
```

You should see output like this, with some differences depending on your OS and hypervisor:

```

> 😄  minikube v1.24.0 on Darwin 12.1

Reviewer 2: Should we make it clear in L106 for the minimum version to use?

Author: Good question.  I can suggest users to use the newer versions of minikube like v1.24.0 but have no idea which minikube version is minimum requirement. I mean users can run TiDB on minikube using default configuration.

Reviewer 2: OK, let's just suggest users to use the newer versions, thanks!

Author: Added another commit to suggest users to use the newer versions.
